plugins/inputs/mqtt_consumer/mqtt_consumer_test.go

```diff
@@ -84,6 +85,10 @@ func (t *FakeToken) SessionPresent() bool {
 	return t.sessionPresent
 }
 
+func (t *FakeToken) Done() <-chan struct{} {
```

Comment:
doesn't appear to be used anywhere. is this for an interface?

Reply:
When I updated the package the tests wouldn't run until I added this